src/crawlee/playwright_crawler/_browser_page_context.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+from crawlee._types import BasicCrawlingContext
+
+if TYPE_CHECKING:
+    from playwright.async_api import Page
+
+
+@dataclass(frozen=True)
+class BrowserPageContext(BasicCrawlingContext):
```

Comment:
This should be exported from `crawlee.playwright_crawler` so that you can easily import it to type your hook parameters.